tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py37, py38, py39, lint
+envlist = py37, py38, py39, lint, typing
 skip_missing_interpreters = True
 
 [gh-actions]
```

Comment:
Please add the `typing` env to line 7 below.

Reply:
Done, but please remark I have not been able to test this (workflow is not activated for my account).